api/kyverno/v1/image_verification_types.go

```diff
@@ -226,9 +226,23 @@ type KeylessAttestor struct {
 
 type CTLog struct {
 	// URL is the address of the transparency log. Defaults to the public log https://rekor.sigstore.dev.
-	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Optional
```

Comment:
In the various attestators, we allow `Rekor` to be `nil. 

What is the behavior when `Rekor` is set to `nil` in an attestor?

The prior behavior is documented in the comments, which may need to be updated.

```go
	// Rekor provides configuration for the Rekor transparency log service. If the value is nil,
	// Rekor is not checked. If an empty object is provided the public instance of
	// Rekor (https://rekor.sigstore.dev) is used.

```

Reply:
When the `Rekor` is set to nil, I am providing https://rekor.sigstore.dev/  as the rekor url https://github.com/kyverno/kyverno/pull/7248/files#diff-4e334a7b5c4dc63f280d7a0e49015d54a381c6445fdc4db8065cc9c9d330e255R504-R506

Reply:
This is the opposite of the prior behavior....and may be confusing. 

Previously setting to `nil` meant that Rekor is not used. Now setting to `nil` means that the public Rekor is used, but if we inspect the policy the URL is not visible.

Perhaps we can make `Rekor` a required attribute and set the URL to the public instance by default, so it will be clear?

This will be a breaking change, so we will need to release note for users with existing policies.